networks/suzuka/suzuka-client/bin/demoscenario.rs

```diff
@@ -0,0 +1,59 @@
+use anyhow::Result;
+use std::sync::Arc;
+/// A simple demo scenario that sleep a few milli second and log some messages.
+/// To run it use: cargo run --release --bin demoscenario
+use suzuka_client::load_soak_testing::execute_test;
+use suzuka_client::load_soak_testing::init_test;
+use suzuka_client::load_soak_testing::ExecutionConfig;
+use suzuka_client::load_soak_testing::Scenario;
+use suzuka_client::load_soak_testing::TestKind;
+
+fn main() {
+	// Define the Test config. Use the default parameters.
+	let mut config = ExecutionConfig::default();
+
+	//define soak test. Remove this line for load test.
+	config.kind = TestKind::Soak {
+		min_scenarios: 6,
+		max_scenarios: 10,
+		duration: std::time::Duration::from_secs(20),
```

Comment:
So for this soak test, we run it for 20 seconds right? Maybe we want a `Duration` enum type here for setting minutes / hours / days.